template.hbr

```diff
@@ -204,11 +210,21 @@ players:
                     "font_style": null,
                     "font_weight": 700
                 },
+                "enum": {
+                    "color": "#{{maroon}}",
+                    "font_style": null,
+                    "font_weight": 700
+                },
                 "function": {
                     "color": "#{{blue}}",
                     "font_style": "italic",
                     "font_weight": null
                 },
+                "hint": {
+                    "color": "#{{maroon}}",
```

Comment:
I think `hint` would come under Information? 

![image](https://github.com/catppuccin/zed/assets/58985301/72ab9f86-f7cd-44be-aa53-4448b874a1f4)

This is what I'm thinking what a `hint` is:

![image](https://github.com/catppuccin/zed/assets/58985301/4dbc4d2d-83d3-44a9-8c52-1a8c660d59a8)

```suggestion
                    "color": "#{{teal}}",
```

Reply:
makes sense to categorize it under Information. i'll make the change